README.md

```diff
@@ -2,46 +2,61 @@
 
 # Strauss – PHP Namespace Renamer
 
-A tool to prefix namespaces and classnames in PHP files to avoid autoloading collisions.
+A tool to prefix namespaces, classnames, and constants in PHP files to avoid autoloading collisions.
 
-A fork of [Mozart](https://github.com/coenjacobs/mozart/). For [Composer](https://getcomposer.org/) for PHP.
+A fork of [Mozart](https://github.com/coenjacobs/mozart/) for [Composer](https://getcomposer.org/) for PHP.
```

Comment:
I was using the punctuation as emphasis. I think both are valid and neither is better.